public/app/ui/page/about.html

```diff
@@ -0,0 +1,7 @@
+<article id="HomePage">
```

Comment:
Actually this made me realise I'd totally missed putting wrapper elements on the view templates, so I've just pushed that change in.